packages/edit-navigation/src/components/header/more-menu.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+import { MenuGroup, MenuItem, VisuallyHidden } from '@wordpress/components';
+import { external } from '@wordpress/icons';
+import { MoreMenuDropdown } from '@wordpress/interface';
+import { __ } from '@wordpress/i18n';
+
+export default function MoreMenu() {
+	return (
+		<MoreMenuDropdown>
+			{ () => (
+				<MenuGroup label={ __( 'Tools' ) }>
+					<MenuItem
+						role="menuitem"
+						icon={ external }
+						href={ __(
+							'https://github.com/WordPress/gutenberg/tree/trunk/packages/edit-navigation/docs/user-documentation.md'
```

Comment:
:wave: I was translating some strings to my native language and ran into this one. I've seen it in use in a couple other places as well, so I'm curious as to why we use `_()` within the `href` attribute.

Reply:
Thanks for pointing that out. It's not needed here, as these docs aren't internationalized.

I'll spin up a PR to remove it.